### PR TITLE
Codesmells fixes

### PR DESCRIPTION
**Pull request by kapoliub**

### CODE REVIEW

**Review by nadiyafomenko — APPROVED** [review]

**Review by VitalikVoicix — APPROVED** [review]

**Review by moran711 — COMMENTED** [review]
Please change naming of all selectors.

File: src/redux/products/products.sagas.js [hunk 1]
```diff
@@ -37,8 +40,8 @@ export function* handleFilteredProductsLoad({ payload: { forSearchBar } }) {
       yield put(setProductsLoading(true));
     }
 
-    const state = yield select((state) => state.Products);
-    const currency = yield select((state) => state.Currency.currency);
+    const state = yield select(StateProductsSelector);
```
Comment (moran711) [inline src/redux/products/products.sagas.js:43]: Make sense to rename this variable to `selectProductsState`

File: src/redux/products/products.sagas.js [hunk 2]
```diff
@@ -37,8 +40,8 @@ export function* handleFilteredProductsLoad({ payload: { forSearchBar } }) {
...
+    const currency = yield select(StateCurrencySelector);
```
Comment (moran711) [inline src/redux/products/products.sagas.js:44]: same

File: src/redux/products/products.sagas.js (hunk 1 above)
Comment (kapoliub) [reply src/redux/products/products.sagas.js]: fixed

File: src/redux/products/products.sagas.js (hunk 2 above)
Comment (kapoliub) [reply src/redux/products/products.sagas.js]: fixed

**Review by moran711 — APPROVED** [review]

**Pull request merged**